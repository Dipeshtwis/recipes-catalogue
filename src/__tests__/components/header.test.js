/* eslint-disable no-unused-vars */
import React from 'react';
import renderer from 'react-test-renderer';
import header from '../../components/Header';

it('renders header component correctly', () => {
  const header = renderer.create(<header />).toJSON();
  expect(header).toMatchSnapshot();
});
