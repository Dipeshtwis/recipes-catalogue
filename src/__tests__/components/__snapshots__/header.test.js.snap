// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders header component correctly 1`] = `<header />`;
